assets/src/edit-story/app/media/media3p/attribution.js

```diff
@@ -110,7 +110,8 @@ export function TenorAttribution() {
   return (
     <a href={tenorUrl} target={'_blank'} rel={'noreferrer'}>
       <AttributionPill>
-        <TenorLogo data-label={__('Powered by Tenor', 'web-stories')} />
+        {__('Powered by', 'web-stories')}
+        <TenorLogo />
```

Comment:
Why was this changed?

Reply:
#7358 is fixing this already

Reply:
Yeah this is a work around for #7358. That PR wasn't an thing when I started ticket 12 days ago.

Reply:
Let's remove this in favor of said PR then.

Reply:
Okay, the e2e tests fail until #7358 is merged. I have marked it as a blocker..

Reply:
Why are they failing? Because of aXe? Then disable the aXe rule until that PR is merged.